lib/rouge/lexers/fortran.rb

```diff
@@ -7,11 +7,11 @@ module Rouge
   module Lexers
     class Fortran < RegexLexer
       title "Fortran"
-      desc "Fortran 95 Programming Language"
+      desc "Fortran 2008 Programming Language"
```

Comment:
Fortran is an ISO standard and generally backwards-compatible, so valid Fortran 90 code is also valid Fortran 2008 code. But I'd be happy to make that change, if needed.